### PR TITLE
chore: add seata sentinel starter & rename tracing starter

### PR DESCRIPTION
**Pull request by conghuhu**
## Brief changelog
- add seata & sentinel starter
- rename tracing starter to `xxx-spring-boot-starter`（After PR merge, I will synchronously modify `dubbo samples`）

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by conghuhu** [comment]
@chickenlj PTAL

File: dubbo-spring-boot/dubbo-spring-boot-starters/dubbo-sentinel-spring-boot-starter/pom.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    </parent>
+
+    <artifactId>dubbo-sentinel-spring-boot-starter</artifactId>
+    <version>${revision}</version>
+    <packaging>jar</packaging>
+    <description>Apache Dubbo Sentinel Spring Boot Starter</description>
+
+    <properties>
+        <sentinel.version>1.8.6</sentinel.version>
+    </properties>
```
Comment (AlbumenJ) [inline dubbo-spring-boot/dubbo-spring-boot-starters/dubbo-sentinel-spring-boot-starter/pom.xml:36]: Can this version be managed in dependencies bom

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-spring-boot/dubbo-spring-boot-starters/dubbo-sentinel-spring-boot-starter/pom.xml (hunk 1 above)
Comment (conghuhu) [reply dubbo-spring-boot/dubbo-spring-boot-starters/dubbo-sentinel-spring-boot-starter/pom.xml]: Modified

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**